infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/wsplugins/brokerphases/OpenshiftBrokerEnvironmentFactory.java

```diff
@@ -41,13 +40,13 @@ public OpenshiftBrokerEnvironmentFactory(
       AgentAuthEnableEnvVarProvider authEnableEnvVarProvider,
       MachineTokenEnvVarProvider machineTokenEnvVarProvider,
       @Named("che.workspace.plugin_broker.init.image") String initBrokerImage,
```

Comment:
Do we still need a separate `init broker`? Maybe `unified` broker can do it (AFAIU clean up plugins folder) for itself?

Reply:
I was thinking about it, but decided not implementing it. It wasn't really needed for the merging meta.yaml and che-plugin.yaml and can also be tackled if we decide to separate config evaluation and real files downloading.
I'll leave init broker for the time being, let's revise it later

Reply:
I'm +1 on leaving the init broker, especially if we're considering looking at not downloading already downloaded plugins (e.g. to avoid rate limiting), but I'm not sure the current direction on that matter.